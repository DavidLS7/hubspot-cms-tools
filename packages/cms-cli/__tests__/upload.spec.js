/*global banjo */

describe('Upload command', () => {
  it('works with help positional', async () =>
    expect(await banjo(['upload', 'help'])).toMatchSnapshot());

  it('works with help option', async () =>
    expect(await banjo(['upload', '--help'])).toMatchSnapshot());
});
