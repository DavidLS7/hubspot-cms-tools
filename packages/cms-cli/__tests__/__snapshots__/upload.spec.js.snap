// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upload command works with help option 1`] = `
"banjo upload <src> <dest>

Upload a folder or file from your computer to the HubSpot CMS

Positionals:
  src   Path to the local file, relative to your current working directory.
                                                             [string] [required]
  dest  Path in HubSpot Design Tools, can be a net new path. [string] [required]

Options:
  --version     Show version number                                    [boolean]
  --debug, -d   set log level to debug                [boolean] [default: false]
  --help        Show help                                              [boolean]
  --config, -c  path to a config file                                   [string]
  --portal, -p  HubSpot portal id or name from config                   [string]
  --mode, -m    write to <draft | publish>                              [string]
"
`;

exports[`Upload command works with help positional 1`] = `
"banjo upload <src> <dest>

Upload a folder or file from your computer to the HubSpot CMS

Positionals:
  src   Path to the local file, relative to your current working directory.
                                                             [string] [required]
  dest  Path in HubSpot Design Tools, can be a net new path. [string] [required]

Options:
  --version     Show version number                                    [boolean]
  --debug, -d   set log level to debug                [boolean] [default: false]
  --help        Show help                                              [boolean]
  --config, -c  path to a config file                                   [string]
  --portal, -p  HubSpot portal id or name from config                   [string]
  --mode, -m    write to <draft | publish>                              [string]
"
`;
